os/net/mac/csma/csma.h

```diff
@@ -48,9 +48,15 @@
 #ifdef CSMA_CONF_SEND_SOFT_ACK
 #define CSMA_SEND_SOFT_ACK CSMA_CONF_SEND_SOFT_ACK
 #else /* CSMA_CONF_SEND_SOFT_ACK */
-#define CSMA_SEND_SOFT_ACK 0
+#define CSMA_SEND_SOFT_ACK 1
```

Comment:
why this change?

Reply:
ooops! separating out a lot of changes on my side here; seems like i staged the wrong change. should be `CSMA_REQUEST_ACK 1` to have same behaviour as before. Fixing.